x-pack/plugins/infra/public/components/asset_details/tabs/overview/alerts_closed_content.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiBadge, EuiToolTip } from '@elastic/eui';
+
+export const AlertsClosedContent = ({ activeAlertCount }: { activeAlertCount?: number }) => {
+  const shouldRenderAlertsClosedContent = typeof activeAlertCount === 'number';
+
+  if (!shouldRenderAlertsClosedContent) {
+    return null;
+  }
```

Comment:
I think it should be the responsibility of the consumer to decide wether they want to render this component or not  instead of having a component that doesn't render anything when no `activeAlertCount` prop has been passed in.

Reply:
Thanks @thomheymann, you are right in general but the idea here is that before we set the `activeAlertCount` we don't want to display any message and the message should be shown once we have the `activeAlertCount` - so when it's a number.